content/docs/CLI/source-git/init.md

```diff
@@ -1,94 +1,124 @@
 ---
-title: "init"
+title: "packit source-git init"
 draft: false
 disableToc: false
 weight: 1
 ---
 
-# source-git init
+# `packit source-git init`
 
 Initialize a repository to serve as a [source-git repo]({{< ref "/docs/source-git" >}}).
 
 The biggest advantage of this command is the functionality to create source-git
 repos. Here's a quick rundown what packit does:
 
- * Pull upstream git history (the default is not pull and use the checked out
-   branch).
-
  * Fetches specfile and additional sources from Fedora or CentOS dist-git.
 
  * Apply downstream patches as commits.
 
  * Add packit config file.
 
 We have [a guide]({{< ref "/docs/source-git/how-to-source-git.md" >}}) on how
-to create source-git repos manually if you want to learn more.
+to create source-git repos manually if you want to understand precise steps packit performs.
 
 ## Prerequisites
 
- * Git repo in the current working directory.
+ * A clone of the upstream repo with checked out branch which matches the base ref.
+
+ * Git ref of the upstream repo used as a base.
+
+ * Clean dist-git repository.
 
 ## Tutorial
 
 As an example, let's create a source-git repo out of
-[tmux](https://github.com/tmux/tmux).
+[systemd](https://github.com/systemd/systemd-stable). The project uses a
+dedicated repository to provide backports for actively maintained stable
+releases.
+
+First step is to clone the respective dist-git repo (we'll use Fedora Rawhide
+in our case):
+
+    $ cd $FEDORA_DIST_GIT_REPOS
+    $ fedpkg clone systemd
+    $ cd systemd
 
-First step is to clone the upstream repo:
+We should now determine the version in the specfile
 
-    $ git clone https://github.com/tmux/tmux
-    $ cd tmux
+    $ rpmspec -q --qf "%{version}" ./systemd.spec
+    249.4
 
-Now we can run the `init` command and get our source-git repo:
+Since systemd-stable prefixes the backport releases with `v`, we know that we want to base our source-git repo on the `v249.4` tag:
 
-    $ packit init --fedora-package tmux
+    $ cd $FEDORA_SOURCE_GIT_REPOS
+    $ git clone https://github.com/systemd/systemd-stable
+    $ cd systemd-stable
+    $ git checkout 'v249.4'
 
-With the `--fedora-package` option we are telling packit to get downstream
-packaging files from Fedora dist-git.
+All the prereqs should be met now! We have the upstream clone with the proper
+tag checked out and the respective dist-git repo. We can now run the `init`
+command:
 
-Yes, it was that simple.
+    $ packit init v249.4 $FEDORA_SOURCE_GIT_REPOS/systemd-stable $FEDORA_DIST_GIT_REPOS/systemd
```

Comment:
```suggestion
    $ packit source-git init v249.4 $FEDORA_SOURCE_GIT_REPOS/systemd-stable $FEDORA_DIST_GIT_REPOS/systemd
```

Reply:
apparently I'm not good at copy-pasting from terminal :D